packages/streamline/src/components/buttons/button-dock/button-dock.spec.tsx

```diff
@@ -1,9 +1,9 @@
 import { fireEvent } from '@testing-library/react-native';
 import React from 'react';
 
-import { renderWithProvider } from '../../../testing/render-with-provider';
 import { ButtonDock } from './button-dock';
 import { ButtonDockProps } from './button-dock.type';
+import { renderWithProvider } from '../../../testing/render-with-provider';
```

Comment:
With the eslint updates, import order makes less sense, I had this issue recently also.

WDYT about in a latter iteration to add a rule to separate "../" and "./" so the organisation is coherent again? If yes, can you create a ticket? Thanks!

Reply:
yes

Reply:
https://www.notion.so/luko/Improve-Eslint-import-order-rules-0333716cc067484587c87d3e1492df08?pvs=4